web/modules/custom/dpl_react_apps/src/Controller/DplReactAppsController.php

```diff
@@ -55,6 +55,8 @@ public function work(string $wid): array {
     return [
       'material' => dpl_react_render('material', [
         'wid' => $wid,
+        'sms-notifications-for-reservations-enabled' =>
+        (int) dpl_library_agency_reservation_sms_notifications_is_enabled(),
```

Comment:
Consider refactoring library agency configuration into a service.

A static function which accesses state/configuration makes it hard to setup unit tests of the controller. An injected service would make this easier.

Another simple approach would be to just let the `DplReactAppsController` access the configuration directly through the `config()` method but that would make the dependency between the two modules soft. I appreciate that it is an actual method.

Reply:
Also I would appreciate a comment regarding why we need to convert a proper type for a value, a boolean, to an integer.